src/common/config/env.validation.ts

```diff
@@ -12,6 +12,7 @@ import {
   Max,
   Min,
   MinLength,
+  ValidateIf,
```

Comment:
Why not to do like [this](https://github.com/lidofinance/polygon-validators-monitoring/blob/master/src/common/config/config.module.ts#L11)?

Reply:
Current version seems more accurate. We disable unnecessary only

Reply:
I don't agree, you basically ignore variables that cause errors while no validation required at all since e2e tests use distinguished implementation of variables passthrough. You've added an extra level of complexity as well because of some "test" environment we can't rely on, actually.

Reply:
@vgorkavenko will update the behaviour in unit tests PR